Dockerfile

```diff
@@ -13,17 +13,37 @@ COPY --from=builder /home/src/main/resources/logback.xml /home
 COPY --from=builder /home/src/main/resources/log4j.properties /home
 COPY --from=builder /home/target/jpo-aws-depositor-jar-with-dependencies.jar /home
 
+# Set the environment variables
+ENV BOOTSTRAP_SERVER={$DOCKER_HOST_IP}:9092
+ENV DEPOSIT_GROUP=$DEPOSIT_GROUP
+ENV DEPOSIT_TOPIC=$DEPOSIT_TOPIC
+ENV DEPOSIT_BUCKET_NAME=$DEPOSIT_BUCKET_NAME
+ENV DEPOSIT_KEY_NAME=$DEPOSIT_KEY_NAME
+ENV K_AWS_ACCESS_KEY_ID=$K_AWS_ACCESS_KEY_ID
+ENV K_AWS_SECRET_ACCESS_KEY=$K_AWS_SECRET_ACCESS_KEY
+ENV K_AWS_SESSION_TOKEN=$K_AWS_SESSION_TOKEN
+ENV K_AWS_EXPIRATION=$K_AWS_EXPIRATION
+ENV HEADER_ACCEPT=$HEADER_ACCEPT
+ENV HEADER_X_API_KEY=$HEADER_X_API_KEY
+ENV API_ENDPOINT=$API_ENDPOINT
+ENV KAFKA_TYPE=$KAFKA_TYPE
+ENV CONFLUENT_TYPE=$CONFLUENT_TYPE
+ENV CONFLUENT_SECRET=$CONFLUENT_SECRET
+
```

Comment:
Is this necessary if we are providing a .env file to be used with docker-compose?

Reply:
These are not necessary and have been removed